lib/travis/logs.rb

```diff
@@ -1,11 +1,16 @@
-require 'travis/logs/config'
-
 module Travis
   def self.config
     Logs.config
   end
 
   module Logs
+    autoload :Aggregate, 'travis/logs/aggregate'
+    autoload :App, 'travis/logs/app'
+    autoload :Config, 'travis/logs/config'
+    autoload :Existence, 'travis/logs/existence'
+    autoload :Receive, 'travis/logs/receive'
+    autoload :Sidekiq, 'travis/logs/sidekiq'
```

Comment:
The last time someone told me to not use `autoload` (which was here at Travis), it was because of some bug in JRuby that may or may not exist anymore.  I like being explicit, too, though, and my reasons for adding `autoload` can usually be summarized as "laziness", so I'll cut it out if ye think it's a good idea.